### PR TITLE
fix(flags): Strip spaces from string lists parsed from flags.

### PR DESCRIPTION
**Pull request by jtk54**

### CODE REVIEW

File: command/applications/application_save.go [hunk 1]
```diff
@@ -20,7 +20,13 @@ func (f *flagStringArray) String() string {
 }
 
 func (f *flagStringArray) Set(value string) error {
-	*f = strings.Split(value, ",")
+	stripped := make([]string, 0)
```
Comment (lwander) [inline command/applications/application_save.go]: Why only for application_save?

Comment (jtk54) [reply command/applications/application_save.go]: Not used for any others at this point, should I just factor these out into a lib at this point?

Comment (lwander) [reply command/applications/application_save.go]: Up to you, was just curious what the pattern was

**Comment by jtk54** [comment]
Factored into a `util` library.

**Pull request merged**